request_a_govuk_domain/settings.py

```diff
@@ -285,3 +286,9 @@
 
 # Only enable S3 storage if it is explicitly enabled or on AWS
 S3_STORAGE_ENABLED = env.bool("S3_STORAGE_ENABLED", default=IS_AWS)
+
+CELERY_BROKER_URL = env.str("CELERY_BROKER_URL", "redis://localhost/0")
```

Comment:
This will need to change on AWS?

Reply:
I think we will need to use some of the values in this?

CELERY_BROKER_URL = env.str("CELERY_BROKER_URL", "redis://localhost/0")
CELERY_BEAT_SCHEDULE_FILENAME = env.str("CELERY_BEAT_SCHEDULE_FILENAME", default="celerybeat-schedule")
CELERY_RESULT_BACKEND = "django-db"
CELERY_RESULT_EXTENDED = True
CELERY_RESULT_EXPIRES = datetime.timedelta(days=14)
CELERY_TASK_DEFAULT_QUEUE = env.str("QUEUE_NAME", "celery")
CELERY_TASK_TRACK_STARTED = True
CELERY_BROKER_TRANSPORT_OPTIONS = env.json("CELERY_BROKER_TRANSPORT_OPTIONS", {})
CELERY_BROKER_CONNECTION_RETRY_ON_STARTUP = True
CELERY_ACKS_LATE = True